blueprints/ember-fullcalendar/index.js

```diff
@@ -2,10 +2,8 @@ module.exports = {
   normalizeEntityName: function() {},
 
   afterInstall: function() {
-    return this.addBowerPackageToProject('fullcalendar').then(() => {
-      return this.addBowerPackageToProject('fullcalendar-scheduler').then(() => {
-        return this.addAddonToProject('ember-cli-moment-shim', '0.6.2');
-      });
+    return this.addBowerPackageToProject('fullcalendar', '^2.7.3').then(() => {
```

Comment:
I was following the example of another module. Since you would normally specify some sort of version dependency when you install a package via `package.json` or `bower.json`, it seemed prudent to do so here. So for example if there were a version 2.0.0 of scheduler, which presumably is v2 because it has breaking changes, this would install an earlier version rather than the potentially breaking one.

Feel free to delete these if you think it's better. I was just trying to follow "best practices" -- whatever they might be.